src/app/zap-templates/zcl/data-model/chip/device-energy-management-mode-cluster.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
```

Comment:
Copyrights need to be updated throughout.

Reply:
Done